app/lib/actions/pulp3/orchestration/content_view_version/export.rb

```diff
@@ -0,0 +1,86 @@
+module Actions
```

Comment:
We moved these `pulp3/orchestration` actions back into the Katello namespace recently - for example, this one moved to `Actions::Katello::ContentViewVersion::Export` - because `pulp3` doesn't make as much sense when there's no more pulp2.  (I think it was @jturel suggestion) But I wouldn't want to see it moved back without a good reason..

Reply:
So the reason i suggested changing this is that we could run:
```
rake test:katello:test:pulp3 
```
to have all the vcr tests against pulp3 get run.  It did this by looking in all the pulp3 paths: 
https://github.com/Katello/katello/blob/master/lib/katello/tasks/test.rake#L158-L159

i discussed with partha about this and suggested we could either move the orchestration here as i've done and the test, or just move the test, and he was more in favor of moving them both.  I'm open to other suggestions

Reply:
I like what was done on this PR - it makes later refactoring easier to have Actions::Katello::ContentViewVersion::Export more or less a generic shell that starts the (as of today, but maybe not some day in the future) pulp3 orchestration off.